docs/deep-learning/deep-ssm.md

```diff
@@ -0,0 +1,121 @@
+## ShapeWorks DeepSSM
+DeepSSM is a convolutional neural network (CNN) model that can provide particles directly from unsegmented images after it has been trained. This documentation provides an overview of the DeepSSM process, for a full explanation see: [DeepSSM: A Deep Learning Framework for Statistical
+Shape Modeling from Raw Images](https://arxiv.org/abs/1810.00111).
+
+The input to the DeepSSM network are unsegmented 3D images of the anatomy of interest and the output are the particle-based shape models. DeepSSM requires training examples of image/PBM pairs which are generated via the traditional Shapeworks grooming and optimization pipeline or otherwise. Once the network has been trained on these examples it can predict the PBM of unseen examples given only images, bypassing the need for labor intensive segmentation, grooming, and optimization parameter tuning. 
```

Comment:
PBM/PDM ? 
Also the first time the abbreviation is declared give the full form, after that you can use it freely/